docs/eni-and-ip-target.md

```diff
@@ -0,0 +1,49 @@
+## `WARM_ENI_TARGET`, `WARM_IP_TARGET` and `MINIMUM_IP_TARGET` 
+
+The AWS VPC CNI has an [IPAM](https://en.wikipedia.org/wiki/IP_address_management) daemon (`aws-node`) running on
+every node that keeps track of all ENIs and IPs attached to the instance. The CNI binary (`/opt/cni/bin/aws-cni`) is
+invoked by the kubelet when a new pod gets added to the node. The binary calls `aws-node` asking for an IP for the pod.
```

Comment:
when a new pod gets added to *or an existing pod removed from* the node

Also, the binary calls `aws-k8s-agent`, not `aws-node`, which is the Daemonset.